appveyor.yml

```diff
@@ -21,6 +21,7 @@ install:
   # Log HTTP requests.
   - npm config set loglevel http
   # Typical npm stuff.
+  - npm install -g npm@3
```

Comment:
I'd rather not add this line. Node 6 comes with npm 3 anyway and I'd like this project to keep working in the defautl Node 4 configuration, i.e. with the built-in npm 2.